lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -223,11 +223,11 @@ states:
         targetJourney: INELIGIBLE
         targetState: INELIGIBLE_SKIP_MESSAGE
 
-  MULTIPLE_DOC_F2F_CHECK_PAGE:
+  MULTIPLE_DOC_F2F_NINO_CHECK_PAGE:
     response:
       type: page
-      context: f2f
-      pageId: page-multiple-doc-check # PYIC-7016: Update so can produce nino event
+      context: nino-only
```

Comment:
Just a thought just `nino` might be better as the context value? `nino-only` could imply that it only has the NINO option on the page 😅

Reply:
Yes makes sense